src/commands/oas2tson.ts

```diff
@@ -159,8 +208,7 @@ const main = () => {
     options.input,
     options.output,
     options.properties,
-    options.excludeDereferencedIds,
-    options.logger
+    options.excludeDereferencedIds
```

Comment:
if someone wants to pass a custom instance of pino, in this way they can't anymore

Reply:
I don't remember removing that, but apparently I did. Fixed.

https://github.com/nearform/openapi-transformer-toolkit/pull/318/commits/941d116acc8c2c1f529e7d7e9e0bce899bd47e14